### PR TITLE
chore: Add an option to print progress only in cli usage

### PR DESCRIPTION
**Pull request by LaunchLee**
This option makes the output cleaner when doing batch conversions.

### CODE REVIEW

File: ffmpeg_progress_yield/__main__.py [hunk 1]
```diff
@@ -40,7 +46,10 @@ def main() -> None:
         for progress in ff.run_command_with_progress():
             print(f"{progress}/100")
 
-    print(ff.stderr)
+    if args.progress_only:
+        print("\x1B[K")
```
Comment (slhck) [inline ffmpeg_progress_yield/__main__.py:50]: What is the purpose of this?

With it enabled, and without it, these are the outputs I get (macOS, Bash):

image

To me it seems that not printing this escape sequence is the better option?

**Review by slhck — CHANGES_REQUESTED** [review]

**Comment by slhck** [comment]
I like this option, thanks for the PR! I just have a question about the escape sequence you inserted, if you could perhaps answer this. Thanks!

**Comment by LaunchLee** [comment]
About this escape sequence there is a concrete description on [this site](https://en.wikipedia.org/wiki/ANSI_escape_code) in Wikipedia. Usually I reference [this site](https://web.archive.org/web/20190624214929/http://www.termsys.demon.co.uk/vtansi.htm) for convenience and simple usage.

**Comment by slhck** [comment]
I know what ANSI escape codes are; my question is more about why you inserted it for this specific case? If you look at my screenshot, you can see that it only appears to add an extra line to the output that is not required. The output looks fine to me when I simply ignore the escape sequence.

**Comment by LaunchLee** [comment]
Oh, I see. I tested it only on Windows machine with PowerShell 7 and Windows Terminal. There is a extra progress output line when not print this escape sequence.

**Comment by LaunchLee** [comment]
Screenshot:
WindowsTerminal_20240916-154420

**Comment by slhck** [comment]
I understand, so maybe we should output this only under Windows then. I merged it with a Windows-only change. If this causes some problems on *nix systems we can remove that condition. Thanks for your contribution!

**Review by slhck — APPROVED** [review]

**Comment by slhck** [comment]
Merged manually in 14a15e381e990f69c09a6f8d960c82a734d7107c

**Pull request closed without merging**